blocks/init/src/Blocks/components/modal/assets/index.js

```diff
@@ -0,0 +1,24 @@
+import domReady from '@wordpress/dom-ready';
+import manifest from './../manifest.json';
+import MicroModal from 'micromodal';
+
+domReady(() => {
```

Comment:
for all frontend js you must use dynamic import. please red more on this link:

https://infinum.github.io/eightshift-docs/docs/basics/dynamic-import